recipes/fragment-insertion/onlyplacements.patch

```diff
@@ -0,0 +1,48 @@
+--- run-sepp.sh	2017-12-07 13:44:13.124241203 -0800
++++ run-sepp-stefan.sh	2017-12-07 13:44:05.076362969 -0800
+@@ -1,7 +1,7 @@
+ #!/bin/bash  
+ 
+ if [ $# -lt 2 ]; then
+-   echo USAGE: $0 "[input fragments file] [output prefix] [optional: -x number-of-cores ] [optional: -A alignment subset size] [optional: -P placement subset size] [optional: any other SEPP argument] [optional: -t filename reference phylogeny] [optional: -a filename reference alignment]
++   echo USAGE: $0 "[input fragments file] [output prefix] [optional: -x number-of-cores ] [optional: -A alignment subset size] [optional: -P placement subset size] [optional: any other SEPP argument] [optional: -t filename reference phylogeny] [optional: -a filename reference alignment] [optional: -n 1 = no tree-, just placements- computation]
+    Optional commands need not be in order. Any SEPP option can also be passed. For example, use
+    -x 8
+    to make SEPP us 8 threads"
+@@ -76,6 +76,10 @@
+ 			t="$2"
+ 			shift # past argument
+ 			;;
++		-n|--noTreeComputation)
++			noTree="$2"
++			shift # past argument
++			;;
+ 		*)
+ 			opts="$opts"" ""$key"" ""$2"
+ 			shift # past argument
+@@ -111,14 +115,20 @@
+ cp $tmp/${name}_placement.json .
```

Comment:
what's the value of $tmp?

Reply:
that's not part of my code but of Siavash's bash wrapper. I figure it is the main temporary working directory which is spawned for all the sub-problems SEPP iterates over.

Reply:
wondering cause it's not declared anywhere in this patch ... please confirm that this is fine and I'll merge as this is my only comment.

Reply:
because it is not part of the patch (it does not have two leading + chars). Thus, it is fine and it gets defined earlier in Siavash's script.

Reply:
Good morning @antgonza If you have a sec, can you give it another shot?